app/controllers/cinema_halls_controller.rb

```diff
@@ -23,7 +26,10 @@ def create
   def edit
```

Comment:
I've did not realized before, but we are not using `edit` method in API-only model. Consider changing this to `update`. Also you will have to change this method in `routes.rb` file I guess